checks/tasks/__init__.py

```diff
@@ -32,6 +33,8 @@ def ub_ctx(self):
             else:
                 self._ub_ctx.add_ta_file(os.path.join(os.getcwd(), settings.DNS_ROOT_KEY))
             self._ub_ctx.set_option("cache-max-ttl:", str(settings.CACHE_TTL * 0.9))
+            self._ub_ctx.set_option("rrset-roundrobin:", "no")
```

Comment:
This will instruct Unbound to always return the same order for the records in an RRSET. I suggested that to maarten because I believe somewhere else in the code there is the assumption that the same order is always returned.